.github/workflows/ar-ci.yml

```diff
@@ -1,4 +1,4 @@
-name: "AR CI"
+name: AR Build
```

Comment:
Not super clear from the PR sorry, why are we renaming this job?
And would we also want to rename the file?

Reply:
I mainly wanted to [give a name to the job](https://github.com/guardian/dotcom-rendering/pull/8904/files#diff-ce88e3c7010a2d04e88f1ffcf90277550327c5c585c9ad2af448886c897069f7R22) because I noticed in the GH UI in the required checks it's called just "build" 

![image](https://github.com/guardian/dotcom-rendering/assets/19683595/b573c77f-a506-4291-9733-ba994059cd65)

I will keep it as "AR CI" and give the same name to the job as it builds and uploads artifacts to Riff-Raff. CI is more accurate.